File: Beecrowd/Beginner/1000 - 1099/1045 - Triangle Types/Python/1045_TriangleTypes.py
a, b, c = map(float, input().split())

if a >= b + c or b >= a + c or c >= a + b:
    print("NAO FORMA TRIANGULO")
elif a * a == b * b + c * c or b * b == a * a + c * c or c * c == a * a + b * b:
    print("TRIANGULO RETANGULO")
elif a * a > b * b + c * c or b * b > a * a + c * c or c * c > a * a + b * b:
    print("TRIANGULO OBTUSANGULO")
elif a * a < b * b + c * c or b * b < a * a + c * c or c * c < a * a + b * b:
    print("TRIANGULO ACUTANGULO")

if a == b == c:
    print("TRIANGULO EQUILATERO")
elif a == b or b == c or a == c:
    print("TRIANGULO ISOSCELES")

File: Beecrowd/Beginner/1000 - 1099/1045 - Triangle Types/C/1045_TriangleTypes.c
#include <stdio.h>

int main() {
    double a, b, c;
    scanf("%lf %lf %lf", &a, &b, &c);

    if (a >= b + c || b >= a + c || c >= a + b) printf("NAO FORMA TRIANGULO\n");
    else if (a * a == b * b + c * c || b * b == a * a + c * c || c * c == a * a + b * b) printf("TRIANGULO RETANGULO\n");
    else if (a * a > b * b + c * c || b * b > a * a + c * c || c * c > a * a + b * b) printf("TRIANGULO OBTUSANGULO\n");
    else if (a * a < b * b + c * c || b * b < a * a + c * c || c * c < a * a + b * b) printf("TRIANGULO ACUTANGULO\n");

    if (a == b && b == c) printf("TRIANGULO EQUILATERO\n");
    else if (a == b || b == c || a == c) printf("TRIANGULO ISOSCELES\n");
    
    return 0;
}
